.eslintrc.json

```diff
@@ -1,14 +1,22 @@
 {
   "extends": "eslint:recommended",
-  "ecmaVersion": 6,
   "env": {
     "mocha": true,
     "node": true,
     "es6": true
   },
   "rules": {
-    "comma-dangle": [2, "always-multiline"],
-    "no-unused-vars": [2, { "vars": "all", "args": "none" }],
+    "comma-dangle": [
```

Comment:
is there any situation where we still use eslint vs. clang or tslint?

Reply:
A few. I'd like to trop those cases down to zero, but we're not there yet I don't think.